src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/Entities/Policy/PolicyDefinitionProperties.cs

```diff
@@ -39,5 +39,11 @@ public class PolicyDefinitionProperties
         /// </summary>
         [JsonProperty(Required = Required.Always)]
         public JObject PolicyRule { get; set; }
+
+        /// <summary>
+        /// The parameters declaration.
+        /// </summary>
+        [JsonProperty(Required = Required.Default)]
+        public JObject Parameters { get; set; }
```

Comment:
Can you please confirm that there are no changes required for other policy cmdlets? i.e.,
1. Get/set/remove policy definition works with and without parameters
2. Get/set/remove policy assignment works with and without parameters